console/console.go

```diff
@@ -305,14 +320,22 @@ func (c *Console) Welcome() {
 		console.log(" datadir: " + admin.datadir);
 	`)
 	// List all the supported modules for the user to call
-	if apis, err := c.client.SupportedModules(); err == nil {
-		modules := make([]string, 0, len(apis))
-		for api, version := range apis {
-			modules = append(modules, fmt.Sprintf("%s:%s", api, version))
+	apis, err := c.client.SupportedModules()
+	if err != nil {
+		if !isMethodNotFoundError(err) {
+			fmt.Fprintln(c.printer)
+			return
 		}
-		sort.Strings(modules)
-		fmt.Fprintln(c.printer, " modules:", strings.Join(modules, " "))
+		apis = apisAll
+		fmt.Fprintln(c.printer, `WARNING: use all modules because "rpc_moduels" does not supported`)
+		fmt.Fprintln(c.printer, `         some api calls will always fail if the module in the node is disabled`)
```

Comment:
```suggestion
		fmt.Fprintln(c.printer, ` API calls associated with specific namespaces may fail if a service node does not open those namespaces.`)
```
Please refrain from using force-push as it seems unable to pick the only new change.